apps/meteor/app/authentication/server/startup/index.js

```diff
@@ -205,7 +205,11 @@ Accounts.onCreateUser(function (options, user = {}) {
 			to: destinations,
 			from: settings.get('From_Email'),
 			subject: Accounts.emailTemplates.userToActivate.subject(),
-			html: Accounts.emailTemplates.userToActivate.html(options),
+			html: Accounts.emailTemplates.userToActivate.html({
+				...options,
+				name: options.name || options.profile?.name,
+				email: options.email || options.profile?.email,
```

Comment:
```suggestion
				email: options.email || user.emails[0].address,
```
This still doesn't work since `options.profile` doesn't have an `email` field. Try using the email stored in the `user` object instead